starter-code/css/main.css

```diff
@@ -1,8 +1,203 @@
-/*
-Colors:
+/* soft-reset */
+* {
+  padding: 0;
+  margin: 0;
+  list-style: none;
+  box-sizing: border-box;
+}
+html {
+  font-size: 16px;
+}
 
-Text: 1A1A1A
-Green: #00B172
-White: #FFF
+/* global cfg */
+body {
+  color: #1A1A1A;
+  font-family: 'Open Sans', 'Roboto', sans-serif;
+  font-weight: 400;
+  line-height: 1.5;
+}
 
-*/
+h1 {
+  text-align: center;
+  font-size: 4em;
+  margin-bottom: 1rem;
+  font-weight: 400;
+  white-space: nowrap;
+}
+
+h2 {
+  display: inline-block;
+  text-align: center;
+  padding: 0.1em 0;
+  font-size: 1.6em;
+  font-weight: 600;
+  white-space: nowrap;
+}
+h3 {
+  font-size: 1.4em;
+  font-weight: 700;
+  text-align: center;
+  padding-bottom: 1em;
+  white-space: nowrap;
+}
+
+/* header and nav-bar */
+
+.std-padding {
+  padding: 1.2em 1.5em;
+}
+
+.std-padding-heading {
+  padding: .5em 1em;
+}
+
+header {
+  height: 60px;
+  background-color: white;
+  position: fixed;
+  z-index: 2;
+  width: 100%;
+}
+
+/* comments: tentativa de alinhar sem flexbox */
+.logo {
+  height: 100%;
+  width: auto;
+  /* float: left; */
+  /* position: relative; */
+}
+
+/* header nav { */
+  /* text-align: center; */
+  /* float: right; */
+  /* margin: 0 */
+/* } */
+
+.flex {
+  height: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+nav ul li {
+  display: inline-block;
+  margin: 0 .8em;
+}
+
+.shift-content-down-after-header {
```

Comment:
Cuidado ao usar a posição do elemento na página ao dar nome na classe. Se o layout mudar um pouco e alguma seção for adicionada antes dessa, o nome dessa classe já não faz mais sentido.